### PR TITLE
Unit Test fun times - debian

### PR DESCRIPTION
**Pull request by DarthHater**
I decided since it's a beautiful Saturday and I've got nothing wilder to do in the great state of Alaska, to write some unit tests for this here project. This PR will be a bit WIP for a bit, I encourage anyone else to add to it!

So far this pull request makes the following changes:
* Adds test package for testing `debian` package
* Adds `ReleaseTest`
* Adds `VersionTest` 
* Adds `ControlFileParserTest`
* Adds `ControlFileTest`
* Adds `control` resource for testing

### CODE REVIEW

File: src/test/java/net/staticsnow/nexus/repository/apt/internal/debian/VersionTest.java [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+  public void toStringTest() {
+    underTest = setUpVersion(TEST_VERSION);
+
+    assertThat(underTest.toString(), is(equalTo(TEST_VERSION)));
+  }
+
+  public Version setUpVersion(final String version) {
+    return new Version(version);
+  }
+}
```
Comment (mpoindexter) [inline src/test/java/net/staticsnow/nexus/repository/apt/internal/debian/VersionTest.java:67]: The Debian version rules are ridiculously complicated.  Perhaps a TODO or comment here about the additional cases that need testing?

**Comment by mpoindexter** [comment]
👍

File: src/test/java/net/staticsnow/nexus/repository/apt/internal/debian/VersionTest.java (hunk 1 above)
Comment (DarthHater) [reply src/test/java/net/staticsnow/nexus/repository/apt/internal/debian/VersionTest.java:67]: Yeah, I saw that, hah. If you have some cases you want tested just let me know! I imagine it's mostly just testing compareTo on a bunch of different cases?

Comment (DarthHater) [reply src/test/java/net/staticsnow/nexus/repository/apt/internal/debian/VersionTest.java:67]: TODO in: 6afd1b8

**Comment by DarthHater** [comment]
@mpoindexter I merged this because I'm going to spend a bit more time today working on some other packages. @jlstephens89 if you or anyone else has feedback let me know and I'll gladly address it.

**Pull request merged**